packages/heartwood-components/components/03-structure/table.scss

```diff
@@ -140,3 +140,14 @@
 	padding-right: spacing('base');
 	padding-left: spacing('base');
 }
+
+.record_table__user_cell {
```

Comment:
yeah it's custom for implementation of the team tables. you think it makes sense to put in web instead?

Reply:
@CarolynSM yeah if it's not directly applicable to the component as it exists in heartwood it should exist in web or the skill since it's essentially modifying the default styles/behavior. Also just a tiny nitpick, we've been using dashes instead of underscores for block/element/modifier names, in this case your class would be `.record-table__user-cell`